data/settings.js

```diff
@@ -1007,22 +1175,6 @@ command.
 The minimum value for this setting is \`1024\` bytes.`
```

Comment:
sieve_editheader_max_header_size type UINT -> SIZE, and default to '2 k'

Reply:
fixed